tests/drucker_dashboard/cleanup.sh

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env bash
 # Delete clusters
+TEST_FILE_DIRECTORY=$(dirname "$0")
+
 minikube delete -p drucker-test1
 minikube delete -p drucker-test2
 
 rm -f ${KUBE_CONFIG_PATH1:-/tmp/kube-config-path1}
 rm -f ${KUBE_CONFIG_PATH1:-/tmp/kube-config-path2}
+rm -f ${TEST_FILE_DIRECTORY}/db.test.sqlite3
```

Comment:
Since `DB_MODE` is set during `python -m unittest`, we cannot find `DB_MODE` from here.
Fortunately since test db name is hard-coded, we can erase test db like here.

Reply:
How about the case `mysql` is in use?

Reply:
Currently we use sqlite when we run a unit test. MySQL is not supported.

Reply:
Ok, then this hard-coding is acceptable.

Reply:
In future if mysql is supported, this may fail which achieves purpose of unit test :-)

Reply:
Thank you!
I agree with you. First of all, we need to activate ‘drop_all()’ method. I’m not sure why this method doesn’t work.